data/json/items/tool_armor.json

```diff
@@ -3746,5 +3746,57 @@
     "revert_to": "ice_utility_exoskeleton_off",
     "weight_capacity_bonus": "190 kg",
     "use_action": { "type": "transform", "menu_text": "Turn off", "msg": "The %s disengages.", "target": "ice_utility_exoskeleton_off" }
+  },
+  {
+    "id": "armored_welding_mask",
+    "repairs_like": "goggles_swim",
+    "looks_like": "welding_mask_crude",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "armored welding mask" },
+    "description": "A modified welding mask, covered in armor steel, with the darkening element removed, and 'УБЕЙ' painted on the front.  Activate to raise the visor.",
```

Comment:
"covered in armor steel"
should probably be one of:
"covered in armored steel"
"covered in steel armor"

Reply:
> "covered in armor steel" should probably be one of: "covered in armored steel" "covered in steel armor"

i disagree, armor steel is a reasonably common term for steel alloys used in armor

Reply:
as far as i can tell, this is a term mostly used outside of the united states. i try to google it and i get repeatedly funneled to "armour steel" (which pretty readily indicates not US in origin, since we don't use a u in that word), and the handful of exceptions are all on product sites for eu companies.

Reply:
We don't have a specific alloy for that and the recipe is using normal steel, so in that case, I would suggest removing armor from it and just making it say "covered in steel"

Reply:
alright, as i also changed other stuff, i changed the wording of the description to be "covered in tempered steel", that work?